user-service/src/main/java/org/java/mentorship/user/domain/mapper/UserContractMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.java.mentorship.user.domain.mapper;
+
+import lombok.NoArgsConstructor;
+import org.java.mentorship.contracts.user.dto.User;
+import org.java.mentorship.user.domain.UserEntity;
+
+@NoArgsConstructor
+public class UserContractMapper {
+
+    public static User userToContract(UserEntity userEntity) {
```

Comment:
Difficult to test / mock in tests
Please make this method non-static, make this class a spring Component and inject it wherever needed

Reply:
I don't believe this should be mocked since it's just some basic operations, but I will make it a bean